test/system/700-play.bats

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Test podman play
+#
+
+load helpers
+
+@test "podman play with stdin" {
+    run_podman play kube - <<< test.yaml
+    run_podman pod rm -f test_pod
+}
+
+@test "podman play" {
+    run_podman play kube test.yaml
```

Comment:
Same, but different reason (`test.yaml` does not exist)

Reply:
fixed